src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -79,7 +79,7 @@
         "symfony/dom-crawler": "<4.4",
         "symfony/http-client": "<4.4",
         "symfony/form": "<5.2",
-        "symfony/lock": "<4.4",
+        "symfony/lock": "<5.3",
```

Comment:
We should additionally update the version constraint in `require-dev`:

```
"symfony/lock": "^5.3"
```

Reply:
Thank you